server/src/test/java/org/opensearch/gateway/remote/RemoteClusterStateServiceTests.java

```diff
@@ -304,13 +304,15 @@ public void testWriteFullMetadataInParallelFailureForIndexMetadata() throws IOEx
             RemoteClusterStateService.IndexMetadataTransferException.class,
             () -> remoteClusterStateService.writeFullMetadata(clusterState, randomAlphaOfLength(10))
         );
+        assertEquals(0, remoteClusterStateService.getRemoteClusterStateStats().getSuccessCount());
```

Comment:
should we assert on failureCount as well?

Reply:
Also test for cleanCount failure metric getting incremented properly.

Reply:
Sure.